dns/records/workspace-variables/register_qa.tfvars.json

```diff
@@ -2,6 +2,14 @@
   "hosted_zone": {
     "register-trainee-teachers.service.gov.uk": {
       "cnames": {
+        "qa": {
+          "target": "d1r6ei57mr48qv.cloudfront.net",
+          "ttl": 300
```

Comment:
This is the default so could be left out?

Reply:
removed ttl:300